curriculum/challenges/english/14-responsive-web-design-22/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/step-033.md

```diff
@@ -1,34 +1,26 @@
 ---
-id: 6193fa3224d7465850407d3d
+id: 61fdaf9ff894b6a084ecdc1b
 title: Step 33
 challengeType: 0
 dashedName: step-33
 ---
 
 # --description--
 
-The `flex-end` setting has also brought your row's description text to the right of the element. To fix this, create a `.name` selector and give it a `width` property of `100%`, and a `text-align` property of `left`.
-
-Because you're using `flex`, this tells your `.name` elements to take up 100% of the remaining available width, which allows the text to move to the left.
+To prevent the text content from overflowing, give your span[class~="sr-only"] ` selector an `overflow` property set to `hidden` and a `white-space` property set to `nowrap`.
```

Comment:
```suggestion
To prevent the text content from overflowing, give your `span[class~="sr-only"]` selector an `overflow` property set to `hidden` and a `white-space` property set to `nowrap`.
```